system/lib/pkgconfig/zlib.pc

```diff
@@ -0,0 +1,6 @@
+Name: zlib
+Description: zlib compression library
+Version: 1.2.13
+
+Requires:
+Libs: -s USE_ZLIB
```

Comment:
Technically you should probably add `Cflags: -sUSE_ZLIB`.   If zlib has not yet been built or installed then it would be needed on the first compile step.

Also,  you remove the space between the `-s` and the `USE_ZLIB` here?